app/src/main/java/com/hedvig/app/feature/insurance/ui/contractcoverage/PerilsAdapter.kt

```diff
@@ -7,9 +7,11 @@ import androidx.fragment.app.FragmentManager
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.RequestBuilder
+import com.google.android.material.card.MaterialCardView
 import com.hedvig.android.owldroid.fragment.PerilFragment
 import com.hedvig.app.BuildConfig
 import com.hedvig.app.R
+import com.hedvig.app.feature.insurance.ui.detail.coverage.PerilBottomSheet
 import com.hedvig.app.util.extensions.isDarkThemeActive
 import com.hedvig.app.util.extensions.view.setHapticClickListener
 import kotlinx.android.synthetic.main.peril_detail.view.*
```

Comment:
ViewBinding

Reply:
This file will be deleted upon feature completion, hence I will do no improving maintenance on it. 🙂